scripts/qa_crowbarsetup.sh

```diff
@@ -1350,6 +1350,22 @@ function custom_configuration()
             if [ -n "$networkingplugin" ] ; then
                 proposal_set_value neutron default "['attributes']['neutron']['networking_plugin']" "'$networkingplugin'"
             fi
+            # For Cloud5+, proposal attribute names changed
+            if iscloudver 5plus; then
```

Comment:
Are we expecting people using mkcloud together with milestone releases? For other features related to Cloud5, we also use ```iscloudver 5plus``` ...